components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -166,14 +166,16 @@ export class GithubApp {
         try {
             const installationId = ctx.payload.installation?.id;
             const cloneURL = ctx.payload.repository.clone_url;
-            const owner = installationId && (await this.findProjectOwner(cloneURL) || (await this.findInstallationOwner(installationId)));
-            if (!owner) {
-                log.info(`Did not find user for installation. Probably an incomplete app installation.`, { repo: ctx.payload.repository, installationId });
+            const installationOwner = installationId ? await this.findInstallationOwner(installationId) : undefined;
```

Comment:
good point! checking...

Reply:
There seem to be a couple of installations per day:
```
SELECT *
	FROM d_b_app_installation AS ai
    WHERE ai.state = 'installed'
    ORDER by ai.creationTime DESC
    LIMIT 200;
```
So I'd expect this to work as expected.

Reply:
There are a lot of entry in `claimed.platform` state, which mark uncomplete registrations. I don't think that is problem; except that we might want to think hope to increase conversion (out of scope here).

Reply:
The current code will happily do prebuilds for those incomplete registrations. With this change all of them will silently no longer see prebuilds.

Reply:
@svenefftinge, thanks for revealing this! it's indeed a bigger issue. now that we're not encouraging users to install the Gitpod app from the GitHub marketplace, but rather go through `/new` there is need to adjust the "find app installer" logic.

Reply:
I just pushed a change which alters `findInstallation` in a way that can be used as intended in this PR, i.e. it will return the latest recorded installation (with a state != uninstalled) which allows to find the installer of a particular app installation. given that we should be able to fix the wrong user account selection.

For completeness, that still isn't our final destination. We want to select any team {member, owner} with proper access to that repository to be able to ready repository details necessary to create a prebuild.